### ISSUE TITLE
Rearrange online tutorials

### ISSUE THREAD

**Issue by dpsanders** [JuliaLang/www.julialang.org#96]
I suggest the online tutorial list be ordered with the latest at the top.
The Intro to Julia tutorials that do not have links should just be removed from the list.

Is this page automatically generated or should I just edit it?

**Comment by ranjanan** [comment]
@dpsanders send a PR [here](https://github.com/JuliaLang/www.julialang.org/blob/master/learning/index.html)

**Comment by ViralBShah** [comment]
Agree. Also we can just link to the youtube page and avoid having to edit this all the time.
